language/move-model/src/builder/module_builder.rs

```diff
@@ -1411,6 +1411,11 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
                 .error(loc, "decreases specification not supported currently");
             return;
         }
+        if matches!(kind, ConditionKind::SucceedsIf) {
+            self.parent
+                .error(loc, "condition kind is not longer supported");
```

Comment:
Drop "longer"?

"condition kind is not supported"

Reply:
Done